src/main/scala/consul/v1/health/HealthRequests.scala

```diff
@@ -18,7 +18,8 @@ object HealthRequests {
 
   implicit private val NodesHealthServiceReads: Reads[NodesHealthService] = Json.reads[NodesHealthService]
 
-  def apply(basePath: String)(implicit executionContext: ExecutionContext): HealthRequests = new HealthRequests {
+  def apply(basePath: String)(implicit executionContext: ExecutionContext, rb: ConsulRequestBasics): HealthRequests = new HealthRequests {
+    import rb._
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://www.codacy.com/app/Codacy/scala-consul/file/1255759090/issues/source?bid=2788492&fileBranchId=2939115#l22)